packages/vkui/src/components/CustomScrollView/CustomScrollBarController.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+export interface CustomScrollBarController {
+  barRef: React.RefObject<HTMLDivElement>;
+  trackerRef: React.RefObject<HTMLDivElement>;
+  trackerVisible: boolean;
+  onResize: () => void;
+  onMove: (e: MouseEvent) => void;
+  onUp: () => void;
+  onScroll: () => void;
+  onDragStart: (e: React.MouseEvent) => void;
```

Comment:
раз шатаем, то можно сразу переименовать методы

```suggestion
  resize: () => void;
  scroll: () => void;
  dragStart: (event: React.MouseEvent) => void;
  dragging: (event: MouseEvent) => void;
  dragEnd: () => void;
```

Reply:
Переименовал